src/server.js

```diff
@@ -40,10 +40,33 @@ const apolloServer = new ApolloServer({
   context,
   introspection: true,
   extensions,
+  debug: true,
   formatError: (error) => {
-    bugsnag.notify(error);
-    // console.error(error.extensions.exception.stacktrace.join('\n'));
-    return error;
+    const productionError = error;
+    const {
+      extensions: {
+        exception: { stacktrace },
+      },
+    } = error;
```

Comment:
So this is pulling `stacktrace` off of the error object. But we had to comment out the old line because there wasn't a stacktrace. So when this is used on line 58 is this going to cause the same problem we had before?

Reply:
it shouldn't because we're in debug mode (a few lines above). but I suppose it wouldn't hurt to add a destructuring default...

Reply:
I like the idea of the destructuring default. Mostly because I checked the apollo server docs and they say that `debug` is defaulted to `true` automagically. 

![Screenshot 2020-01-15 11 36 59](https://user-images.githubusercontent.com/3229463/72452341-64d6c500-378b-11ea-8323-043de089d805.png)

That just makes me not convinced that this wouldn't have the same problem. Adding the default should at least make it so that it won't cause any problems. 👍

Reply:
love it